bin/gazelle

```diff
@@ -17,11 +17,11 @@ fi
 find . -type f \( -name BUILD -o -name BUILD.bazel \) -print0 | \
     xargs -0 sed "${params[@]}" \
           -e 's|google/rpc:go_default_library|:google/rpc|g' \
-          -e 's|@io_istio_api//mixer/v1/config/descriptor:go_default_library|@io_istio_api//:mixer/v1/config/descriptor|g' \
-          -e 's|@io_istio_api//mixer/v1:go_default_library|@io_istio_api//:mixer/v1|g' \
+          -e 's|@io_istio_api//mixer/v1/config/descriptor:descriptor|@io_istio_api//:mixer/v1/config/descriptor|g' \
+          -e 's|@io_istio_api//:mixer/v1|@io_istio_api//:mixer/v1|g' \
           -e 's|@io_istio_api//mixer/v1/config/client:go_default_library|@io_istio_api//:mixer/v1/config/client|g' \
           -e 's|@io_istio_api//mixer/v1/template:go_default_library|@io_istio_api//:mixer/v1/template|g' \
-          -e 's|@io_istio_api//proxy/v1/config:go_default_library|@io_istio_api//:proxy/v1/config|g'\
-          -e 's|@io_istio_api//broker/v1/config:go_default_library|@io_istio_api//:broker/v1/config|g'\
+          -e 's|@io_istio_api//proxy/v1/config:go_default_library|@io_istio_api//proxy/v1/config:go_default_library|g'\
+          -e 's|@io_istio_api//broker/v1/config:go_default_library|@io_istio_api//broker/v1/config|g'\
```

Comment:
Looks like we should be able to delete most (all?) of these substitutions now that we're using the default generated targets?

Reply:
why did we added this subs ?